### PR TITLE
Move image from docker.io to quay.io

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Comment by jpopelka** [comment]
Already deployed to [the cluster](https://ocp4.psi.redhat.com)

File: Containerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
-ARG base_image=centos:8
+ARG base_image=quay.io/app-sre/centos:8
```
Comment (TomasTomecek) [inline Containerfile:4]: are we sure this image is going to be up to date?

**Review by TomasTomecek — APPROVED** [review]
LGTM

it's really weird to me that we don't push our base images (Red Hat) to our container registry

File: Containerfile (hunk 1 above)
Comment (jpopelka) [reply Containerfile:4]: [The repo](https://quay.io/repository/app-sre/centos?tab=info) says "Mirror of the upstream centos repository" and it was updated a week ago so "hopefully" is the only answer I have.

**Pull request merged**